### PR TITLE
157 link inputs and actions

### PR DESCRIPTION
**Pull request by osvegn**
THe player can move on the screen and shoot buy pressing keyboard

### CODE REVIEW

File: libs/SFMLSystems/InputManagement.cpp [hunk 1]
```diff
@@ -34,28 +34,37 @@ void InputManagement::run(World &world)
...
+        }
         if (event.type == sf::Event::MouseButtonPressed) {
             auto mouseButtonPressed = [event](std::shared_ptr<ecs::Entity> entityPtr) {
                 if (entityPtr->getComponent<MouseInputComponent>().MouseMapActions.contains(event.mouseButton.button)
-                    && entityPtr->contains<AllowMouseAndKeyboardComponent>())
-                    entityPtr->getComponent<ActionQueueComponent>().actions.push(
-                        entityPtr->getComponent<MouseInputComponent>().MouseMapActions[event.mouseButton.button].first);
+                    && entityPtr->contains<AllowMouseAndKeyboardComponent>()) {}
+                    // entityPtr->getComponent<ActionQueueComponent>().actions.push(
+                    //     entityPtr->getComponent<MouseInputComponent>().MouseMapActions[event.mouseButton.button]);
```
Comment (LucasTesnier) [inline libs/SFMLSystems/InputManagement.cpp:56]: Comment ?

**Review by LucasTesnier — CHANGES_REQUESTED** [review]
Just a question

File: libs/SFMLSystems/InputManagement.cpp (hunk 1 above)
Comment (31Nathan) [reply libs/SFMLSystems/InputManagement.cpp:56]: (I'm Thomas) I can remove them, but I will reimplement it in part 2 because it will be more efficient

Comment (LucasTesnier) [reply libs/SFMLSystems/InputManagement.cpp:56]: Np

**Review by LucasTesnier — APPROVED** [review]

**Review by 31Nathan — APPROVED** [review]
NTR

**Pull request merged**